src/io/flutter/run/bazelTest/FlutterBazelTestConfigurationEditorForm.form

```diff
@@ -83,9 +61,36 @@
       </component>
       <vspacer id="3f5de">
         <constraints>
-          <grid row="5" column="0" row-span="1" col-span="1" vsize-policy="6" hsize-policy="1" anchor="0" fill="2" indent="0" use-parent-layout="false"/>
+          <grid row="6" column="0" row-span="1" col-span="1" vsize-policy="6" hsize-policy="1" anchor="0" fill="2" indent="0" use-parent-layout="false"/>
         </constraints>
       </vspacer>
+      <component id="bff6f" class="javax.swing.JLabel">
+        <constraints>
+          <grid row="1" column="1" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+        </constraints>
+        <properties>
+          <enabled value="false"/>
+          <text value="The absolute entry-point path for the application. Generally, something like /.../lib/main_test.dart."/>
+        </properties>
+      </component>
+      <component id="8e5b7" class="javax.swing.JLabel">
+        <constraints>
+          <grid row="3" column="1" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+        </constraints>
+        <properties>
+          <enabled value="false"/>
+          <text value="The absolute path to a Bazel launching script."/>
+        </properties>
+      </component>
+      <component id="eff55" class="javax.swing.JLabel">
+        <constraints>
+          <grid row="5" column="1" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+        </constraints>
+        <properties>
+          <enabled value="false"/>
+          <text value="The Bazel target, some //a/b/c:name."/>
```

Comment:
Here, we should say `Generally, something like...` for similarity with above, or use an e.g.

Reply:
Yes, I'll be following up to make the two bazel launch configs more consistent. I went back and forth on the "Generally".